### PR TITLE
Adding curl examples

### PR DESCRIPTION
**Pull request by leflambeur**
Added curl examples from the old Read The Docs to the API Reference, largely should be ported with proper formatting applied and any old Product References to Archivist or similar updated

### CODE REVIEW

File: content/docs/api-reference/assets-api/index.md [hunk 1]
```diff
@@ -13,4 +13,171 @@ weight: 101
...
 ---
 
+## Asset API Examples
+
+Create the [bearer_token](../../setup-and-administration/getting-access-tokens-using-client-secret) and store in a file in a secure local directory with 0600 permissions.
+
+Set the URL (for example):
+
+```bash
+export URL=https://synsation.1234-5678.nodes.archivist.jitsuin.io
```
Comment (JAG-UK) [inline content/docs/api-reference/assets-api/index.md:23]: This should be “https://app.rkvst.io” in all the examples

File: content/docs/api-reference/assets-api/index.md [hunk 2]
```diff
@@ -13,4 +13,171 @@ weight: 101
...
+        ]
+    },
+    "confirmation_status": "PENDING",
+    "tracked": "TRACKED"
+}
+```
+
+### Asset Record Retrieval
+
+Asset records in Jitsuin Archivist are tokenized at creation time and referred to in all API calls and smart contracts throughout the system by a unique identity of the form:
```
Comment (JAG-UK) [inline content/docs/api-reference/assets-api/index.md:96]: All instances of “Jitsuin Archivist” or “Archivist” need to be changed to RKVST.

This should be an easy search-and-replace but be careful not to change any of the URLs or resource identifiers which do still use ‘archivist’

File: content/docs/api-reference/tls-ca-certificates-api/index.md [hunk 3]
```diff
@@ -13,4 +13,158 @@ weight: 113
...
+
+```json
+{
+    "display_name": "Some description",
+    "certificate": "-----BEGIN CERTIFICATE-----\nMIIFxDCCA6ygAwIBAgIBAjANBgkqhkiG9w0BAQsFADCBsDELMAkGA1UEBhMCVVMx\nETAPBgNV....1NF/BjDZ4wdexw==\n-----END CERTIFICATE-----\n"
+}
+```
+
+To include the PEM file content in a JSON string it must be flattened to a single line. 
+To create a single line representation of a PEM file for the archivist api, you must replace new lines with the literal string “n”.
```
Comment (JAG-UK) [inline content/docs/api-reference/tls-ca-certificates-api/index.md:38]: It’s \n not just n!

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: content/docs/api-reference/assets-api/index.md (hunk 1 above)
Comment (leflambeur) [reply content/docs/api-reference/assets-api/index.md]: All should be updated

File: content/docs/api-reference/assets-api/index.md (hunk 2 above)
Comment (leflambeur) [reply content/docs/api-reference/assets-api/index.md]: All should be updated, barring the API endpoints

File: content/docs/api-reference/tls-ca-certificates-api/index.md (hunk 3 above)
Comment (leflambeur) [reply content/docs/api-reference/tls-ca-certificates-api/index.md]: Fixed :) - that was a direct copy form the old docs!

File: content/docs/api-reference/assets-api/index.md (hunk 2 above)
Comment (JAG-UK) [reply content/docs/api-reference/assets-api/index.md]: There's one left, but I'll address that in my PR for the tagline since it's in the same area

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**